doc/sphinx-guides/source/admin/metadatacustomization.rst

```diff
@@ -76,453 +76,308 @@ Each of the three main sections own sets of properties:
 #metadataBlock properties
```

Comment:
Even if I hide whitespace changes it's difficult to tell what's changing in this table. Can you please edit the description of the pull request and give a few hints?

Reply:
@j-n-c updated the description (thanks). Some URLs in the table were not clickable.